python/ml4ir/base/model/architectures/fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,37 @@
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class FixedAdditivePositionalBias(layers.Layer):
+    """
```

Comment:
@mohazahran @lastmansleeping this class is the gist of this PR/contribution. And this trick maybe quite powerful in reality. But notice how cryptic/absent the description is although the code makes many assumptions! You should take 15' and actually describe what it does I think (if not for any other reason, so that you can brag about it!). I am exaggerating on purpose a bit..

```
Implements a solution to handle positional bias in logged data by adding a bias term to the relevance score of a document-query pair. We expect the bias term (learned during training) to be higher for higher ranks. Example: 
Assume 3 doc-query pairs as follows: 

... you give your example here and you show that something is added somewhere ....
  
The process of adding the positional bias is only applied during training because this is a way to de-bias data from logs (typically clicked/no clicked pairs). During inference we do not know the actual ranks, so the addition is turned off. 

This is an approach we have found useful for logged data as these are presented to the user with some order and there is a presentation bias (that this technique tries to model). When having actual relevance annotations you should probably not use this technique. 

Reference: a paper/blog if one exists
```
This is an example of how I would go with this.

Reply:
Great suggestions @balikasg. I updated the PR and took them into account.

Reply:
thanks! you skipped the example.. 
Nice implementation btw!